beacon-chain/state/v1/field_trie_helpers.go

```diff
@@ -27,8 +27,8 @@ func validateElements(field fieldIndex, elements interface{}, length uint64) err
 	return nil
 }
 
-// this converts the corresponding field and the provided elements to the appropriate roots.
-func fieldConverters(field fieldIndex, indices []uint64, elements interface{}, convertAll bool) ([][32]byte, error) {
+// convertToRoots converts the corresponding field and the provided elements to the appropriate roots.
+func convertToRoots(field fieldIndex, indices []uint64, elements interface{}, convertAll bool) ([][32]byte, error) {
```

Comment:
don't think `convertToRoots` is a particularly good name for what this method actually does, it is too generic a name here.